src/webview/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -389,10 +431,16 @@ public boolean getLocationPermissions() {
 	@Override
 	public void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
 		super.onRequestPermissionsResult(requestCode, permissions, grantResults);
-		if (requestCode != ACCESS_FINE_LOCATION_PERMISSION_REQUEST_CODE) {
+
+		if (requestCode == ACCESS_FINE_LOCATION_PERMISSION_REQUEST_CODE) {
+			locationRequestResolved();
+			return;
+		}
+
+		if (requestCode == ACCESS_STORAGE_PERMISSION_REQUEST_CODE) {
```

Comment:
We're only adding this for the debug message?

Reply:
Fixed